aladdin/lib/logging.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+import lazy_object_proxy
+
+
+def getLogger(*args, **kwargs):
+    return lazy_object_proxy.Proxy(lambda: logging.getLogger(*args, **kwargs))
```

Comment:
used so that any `logger` instances created at the module level are lazily evaluated and get an instance of `VerboseLogs`